roles/servicetelemetry/tasks/component_alertmanager.yml

```diff
@@ -88,7 +117,7 @@
       roleRef:
         apiGroup: rbac.authorization.k8s.io
         kind: ClusterRole
-        name: alertmanager-main
+        name: alertmanager-stf
```

Comment:
Kubernetes won't let you change an existing roleRef in a ClusterRoleBinding, so we'll actually need to remove this ClusterRoleBinding and re-create it with a new roleRef. I'm going to implement a potential fix and test it locally.